### PR TITLE
open prev/next labeled image

### PR DESCRIPTION
**Pull request by JiamingJiao**
Press Q/E (Ctrl+Shift+Q/E) to go to previous/next labeled image. It is useful when you are reviewing a discontinuously labeled sequence of images.

### CODE REVIEW

File: anylabeling/configs/anylabeling_config.yaml [hunk 1]
```diff
@@ -91,6 +91,8 @@ shortcuts:
 
   open_next: [D, Ctrl+Shift+D]
   open_prev: [A, Ctrl+Shift+A]
+  open_next_labeled: [E, Ctrl+Shift+E]
```
Comment (CVHub520) [inline anylabeling/configs/anylabeling_config.yaml:94]: hotkey conflict

**Comment by JiamingJiao** [comment]
I changed the shortcuts to Z/C (Ctrl+Shift+Z/C) and added icons for the buttons.

**Comment by CVHub520** [comment]
Hi @JiamingJiao,

Thank you for your contribution to our project. We appreciate your effort and interest in improving our software.

I have reviewed your pull request, and there are a few changes we would like you to make to ensure it meets our project standards. Additionally, it seems there are some merge conflicts that need to be resolved.

Could you please address the feedback provided and resolve the merge conflicts? Once you’ve made the necessary changes, feel free to update the pull request, and we will be happy to review it again.

If you need any assistance or have any questions, please don’t hesitate to reach out. We are here to help.

Looking forward to your updated submission.

**Comment by CVHub520** [comment]
There are a couple of adjustments we need to make to ensure it aligns with our project requirements:
1. Please use the [Ctrl+Shift+D] and [Ctrl+Shift+A] keyboard shortcuts to trigger the desired functionality.
2. You may need to remove all relevant icons from the interface as part of this update.

**Review by CVHub520 — CHANGES_REQUESTED** [review]
There are a couple of adjustments we need to make to ensure it aligns with our project requirements:

Please use the [Ctrl+Shift+D] and [Ctrl+Shift+A] keyboard shortcuts to trigger the desired functionality.
You may need to remove all relevant icons from the interface as part of this update.

**Comment by CVHub520** [comment]
The current PR has been implemented in the latest [#8f5d0e6](https://github.com/CVHub520/X-AnyLabeling/commit/8f5d0e678272a2c478d52f9e9d1f545f72582503) branch and will be closed. Thank you again for your contribution!

**Pull request closed without merging**